<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogFormatWidget</class>
 <widget class="QWidget" name="LogFormatWidget">
  <property name="windowTitle">
   <string>Format</string>
  </property>
  <property name="windowIcon">
   <iconset theme="object-columns"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Log format:</string>
       </property>
       <property name="buddy">
        <cstring>logFormatComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="logFormatComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="addFormatButton">
         <property name="text">
          <string>+</string>
         </property>
         <property name="icon">
          <iconset theme="list-add">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Parser:</string>
       </property>
       <property name="buddy">
        <cstring>parserLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="parserLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="highlightListView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="HighlightWidget" name="highlightWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HighlightWidget</class>
   <extends>QWidget</extends>
   <header>HighlightWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>logFormatComboBox</tabstop>
  <tabstop>addFormatButton</tabstop>
  <tabstop>highlightListView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
